l3build-upload.lua

```diff
@@ -248,11 +254,13 @@ function construct_ctan_post(uploadfile,debug)
   ctan_field("uploader",     uploadconfig.uploader,      255, "Name of uploader",                    true,  false )
   ctan_field("version",      uploadconfig.version,        32, "Package version",                     true,  false )
 
+  --[=[ `qq` is unused because line 258 is commented
```

Comment:
line 256/264 instead of line 258
This line # is subject to change

Reply:
I think this should just be removed (including 256/264). If we ever need it again we can try using one of these fancy version control systems to get it back.